aspnet/fundamentals/owin.rst

```diff
@@ -1,8 +1,386 @@
+	* - owin.RequestProtocol
+	  - ``String``
+	  -
+	* - owin.RequestHeaders
+	  - ``IDictionary<string,string[]>``
+	  -
+	* - owin.RequestBody
+	  - ``Stream``
+	  - 
+	 
+Request Data (OWIN v1.1.0)
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. list-table::
+	:header-rows: 1
+	
+	* - Key
+	  - Value (type)
+	  - Description
+	* - owin.RequestId
+	  - ``String``
+	  - Optional
+
+Response Data (OWIN v1.0.0)
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. list-table::
+	:header-rows: 1
+	
+	* - Key
+	  - Value (type)
+	  - Description
+	* - owin.ResponseStatusCode
+	  - ``int``
+	  - Optional
+	* - owin.ResponseReasonPhrase
+	  - ``String``
+	  - Optional
+	* - owin.ResponseProtocol
+	  - ``String``
+	  - Optional
+	* - owin.ResponseHeaders
+	  - ``IDictionary<string,string[]>``
+	  - 
+	* - owin.ResponseBody
+	  - ``Stream``
+	  -
+
+Other Data (OWIN v1.0.0)
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. list-table::
+	:header-rows: 1
+	
+	* - Key
+	  - Value (type)
+	  - Description
+	* - owin.CallCancelled
+	  - ``CancellationToken``
+	  -
+	* - owin.Version
+	  - ``String``
+	  -
+
+IAppBuilder.Properties
+^^^^^^^^^^^^^^^^^^^^^^
+
+.. list-table::
+	:header-rows: 1
+	
+	* - Key
+	  - Value (type)
+	  - Description
+	* - builder.AddSignatureConversion
+	  -
+	  -
+	* - builder.DefaultApp
+	  -
+	  -
+
+Common Keys
+^^^^^^^^^^^
+
+.. list-table::
+	:header-rows: 1
+	
+	* - Key
+	  - Value (type)
+	  - Description
+	* - ssl.ClientCertificate
+	  - ``X509Certificate``
+	  -
+	* - ssl.LoadClientCertAsync
+	  - ``Func<Task>``
+	  -
+	* - server.RemoteIpAddress
+	  - ``String``
+	  -
+	* - server.RemotePort
+	  - ``String``
+	  -
+	* - server.LocalIpAddress
+	  - ``String``
+	  -
+	* - server.LocalPort
+	  - ``String``
+	  -
+	* - server.IsLocal
+	  - ``bool``
+	  -
+	* - server.Capabilities
+	  - ``IDictionary<string,object>``
+	  -
+	* - server.OnSendingHeaders
+	  - ``Action<Action<object>,object>``
+	  -
+	* - host.TraceOutput
+	  - ``TextWriter``
+	  - Available to the application for low volume debug statements
+	* - host.Addresses
+	  - ``IList<IDictionary<string,object>>``
+	  -
+	* - host.AppName
+	  - ``String``
+	  - A name describing the application instance.
+	* - host.OnAppDisposing
+	  - ``CancellationToken``
+	  - Signaled when the application is preparing to shut down
+	* - host
+	  - ``String``
+	  - The host portion of the URI, e.g. `docs.asp.net`
+	* - scheme
+	  - ``String``
+	  - The scheme portion of the URI, e.g. `http`
+	* - port
+	  - ``String``
+	  - The port portion of the URI, e.g. `80`
+	* - path
+	  - ``String``
+	  - The path portion of the URI, e.g. `/en/latest/fundamentals/owin.html`
+
+SendFiles v0.3.0
+^^^^^^^^^^^^^^^^
+
+.. list-table::
+	:header-rows: 1
+	
+	* - Key
+	  - Value (type)
+	  - Description
+	* - sendfile.Version
+	  - ``String``
+	  -
+	* - sendfile.Support
+	  - ``IDictionary<string,object>``
+	  -
+	* - sendfile.Concurrency
+	  - ``String``
+	  - "Overlapped" indicates support for multiple outstanding async operations.
+
+Opaque v0.3.0
```

Comment:
For Opaque? Should I bother, since it doesn't appear to be supported by OwinContext?